### ISSUE TITLE
ask about learning rate

### ISSUE THREAD

**Issue by wfz666** [rayabhisek123/CFAT#14]
Hi, I am now using 4 A100 to train the model, but it is really slow(5 days) for batch size 32, so I would change the batch size to bigger and also the learning rate and milestone. So can you give me suggestions? Thanks!

**Comment by Emmlll** [comment]
> Hi, I am now using 4 A100 to train the model, but it is really slow(5 days) for batch size 32, so I would change the batch size to bigger and also the learning rate and milestone. So can you give me suggestions? Thanks!


Dear sir, Could you please provide the download link for the pre-trained models? I would like to test model on my dataset,thank you very very much.email：:wyt18436070526@163.com
